sdcm/cluster.py

```diff
@@ -3026,23 +3042,26 @@ def install_scylla_monitoring_prereqs(self, node):
                 pip install pyyaml
             """)
         elif node.is_debian():
```

Comment:
is_debian() will be True for debian 8 & debian 9,  but you added some code for jessie only

Reply:
scylla-manager doesn't support debian-9. so all testing is only for Debian-8 currently.